sysutils/tmux/Makefile

```diff
@@ -0,0 +1,67 @@
+# Created by: Wen Heping <wenheping@gmail.com>
+
+PORTNAME=	tmux
+PORTVERSION=	3.2
+CATEGORIES=	sysutils
+MASTER_SITES=	https://github.com/tmux/tmux/releases/download/${PORTVERSION}/
+DISTFILES=	${DISTNAME}${EXTRACT_SUFX}
+
+MAINTAINER=	kraileth@elderlinux.org
+COMMENT=	Terminal Multiplexer
+
+LICENSE=			iscl gpl2
+LICENSE_COMB=			multi
+LICENSE_FILE_ISCL=		${WRKSRC}/COPYING
+LICENSE_FILE_GPL2=		${WRKSRC_bash}/COPYING
+LICENSE_DISTFILES_ISCL=		${DISTNAME}${EXTRACT_SUFX}
+LICENSE_DISTFILES_GPLv2=	${DISTFILE_bash}
+
```

Comment:
The LICENSE_FILE_ and LICENSE_DISTFILES lines should use the same case for license names.  So you'd want LICENSE_FILE_gpl2 for instance. 

You can use make license-list  to see the list of licenses. 

I also recommend setting MPORT_MAINTAINER_MODE="YES" in /etc/make.conf so you can see errors with plist entries when developing ports.